Constitution.md

```diff
@@ -77,16 +77,20 @@ Membership of the club is Free and lasts for a full academic year, September to
 
 CSSA will consist of the following executives members, who shall be voting members of the group as part of the council:
 
-- President(s)
-- Vice-President Student Affairs
-- Vice-President Internal
-- Vice-President Lounge Affairs
-- Vice-President Events
-- Interclub Liaison
+- Co-Presidents
+- Director of Finances and Internal Affairs
```

Comment:
```suggestion
- Executive of Finances and Internal Affairs
- Executive of Lounge Affairs
- Executive of Student Affairs
- Executive of Events
- Executive of Advocacy
```

Executive is a defined term with a meaning, while Director doesn't mean anything.

Reply:
The purpose is the standardized one that has been used for the class presentations along with wics and .devClub

noun
noun: director; plural noun: directors

    a person who is in charge of an activity, department, or organization.
    "he has been appointed finance director"
        a member of the board of people that manages or oversees the affairs of a business.
        
We can use other words, don't have to use director as other synonyms exist. Although some previous motions have already been signed as such, and Executive itself refers to the executive body as a whole but not specific positions. 

Requires more discussion to finalize.